agent/OffshoredediBuyer.py

```diff
@@ -21,25 +21,14 @@ class OffshoredediBuyer(VPSBuyer):
     """
     This class orders a VPS from offshorededi.com
     """
-    def __init__(self, email = "", password = ""):
+    def __init__(self, email="", password=""):

```

Comment:
I agree but python styling for some reason does not. Look at https://docs.python.org/3/tutorial/controlflow.html#default-argument-values or http://www.diveintopython.net/power_of_introspection/optional_arguments.html for examples.